### PR TITLE
chore(wdio): porting the scoped-basic test suite to wdio

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
Porting the scoped-basic test suite to wdio!

## What is the current behavior?

GitHub Issue Number: #5458 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Testing

We live and die by CI.

### CODE REVIEW

File: test/wdio/global.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { setMode } from '@stencil/core';
+
+const globalScript = () => {
+  setMode((elm) => {
+    return (elm as any).colormode || elm.getAttribute('colormode') || (window as any).KarmaMode;
```
Comment (christian-bromann) [inline test/wdio/global.ts:5]:
```suggestion
    return (elm as any).colormode || elm.getAttribute('colormode');
```

File: test/wdio/scoped-basic/cmp.test.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+import exp from 'constants';
```
Comment (christian-bromann) [inline test/wdio/scoped-basic/cmp.test.tsx:3]:
```suggestion
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]

File: test/wdio/scoped-basic/cmp.test.tsx (hunk 2 above)
Comment (alicewriteswrongs) [reply test/wdio/scoped-basic/cmp.test.tsx:3]: oh I didn't even see that! wonder why it was there 🤔

File: test/wdio/global.ts (hunk 1 above)
Comment (alicewriteswrongs) [reply test/wdio/global.ts:5]: ah makes sense :)

Comment (alicewriteswrongs) [reply test/wdio/global.ts:5]: gonna be honest, I moved this file without reading it that closely :O

Comment (christian-bromann) [reply test/wdio/global.ts]: haha .. been doing that too 🙈

**Review by christian-bromann — APPROVED** [review]

File: test/wdio/scoped-basic/cmp.test.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { h, Fragment } from '@stencil/core';
```
Comment (rwaskiewicz) [inline test/wdio/scoped-basic/cmp.test.tsx:1]: Non-blocking - `Fragment` isn't used and can be removed
```suggestion
import { h } from '@stencil/core';
```

**Review by rwaskiewicz — APPROVED** [review]

**Pull request merged**